modules/configuring-monitoring-for-the-multi-model-serving-platform.adoc

```diff
@@ -4,7 +4,9 @@
 = Configuring monitoring for the multi-model serving platform
 
 [role="_abstract"]
-The multi-model serving platform includes metrics for the ModelMesh component. When you have configured monitoring, you can grant Prometheus access to scrape the available metrics.
+The multi-model serving platform includes per model and model server metrics for the ModelMesh component. ModelMesh generates its own metrics and does not rely on the underlying serving runtime to provide them. The set of available metrics include metrics for model request rates and timings, model loading and unloading rates, times and sizes, internal queuing delays, capacity and usage, cache state, and least recently used models. For more information, see link:https://github.com/kserve/modelmesh-serving/blob/main/docs/monitoring.md[ModelMesh metrics]. 
```

Comment:
Either "per-model" or just "model". I think I prefer "model".

Reply:
@syaseen-rh I don't see this reflected in this latest version. Per the comment above, I don't think "per model" is correct - we should change to "per-model" or just "model".

Reply:
Wow, I was sure I had updated "per model" to "model". I'll fix this right now, thanks for the approval :)